core/bridges/cache.go

```diff
@@ -0,0 +1,269 @@
+package bridges
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+	"sync"
+	"time"
+
+	"golang.org/x/exp/maps"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
+	"github.com/smartcontractkit/chainlink-common/pkg/sqlutil"
+
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	"github.com/smartcontractkit/chainlink/v2/core/utils"
+)
+
+const (
+	CacheServiceName         = "BridgeCache"
+	DefaultUpsertInterval    = 500 * time.Millisecond
+	DefaultBulkInsertTimeout = 250 * time.Millisecond
```

Comment:
How were these values chosen?

Reply:
Pin the tail on the donkey. Lol. Do you have some thoughts on what they should be? I was actually basing the timeout on being more than a standard query, though I don't know what qualifies as a long running query within this system. The upsert interval would need to be longer than the timeout and be regular enough to ensure we have a sensible backup, but not run so often than we overload the database again.

Reply:
Are we going to wish these were configurable later on? Maybe we should expose one of them, and then derive the other from that.